apis/management.cattle.io/v3/cluster_scan_types.go

```diff
@@ -133,19 +133,19 @@ type CisBenchmarkVersion struct {
 
 type ScheduledClusterScanConfig struct {
 	// Cron Expression for Schedule
-	Schedule string `yaml:"schedule" json:"schedule,omitempty"`
+	CronSchedule string `yaml:"cronSchedule" json:"schedule,omitempty"`
```

Comment:
json tag also needs to be updated to `cronSchedule`

Reply:
Thank you, will update.